### ISSUE TITLE
target_border property

### ISSUE THREAD

**Issue by halilbilgin** [catboost/catboost#1165]
Hello,

I would like to use probabilistic labels in my binary classification problem. Specifically, my aim is to compute the loss function based on the smooth probabilistic labels during training.

My question is: When I set target_border property, which I have to, will Catboost use probabilistic continous labels between 0 and 1 during computing the loss function (i.e. logloss in my case) ?

Thanks,

**Comment by annaveronika** [comment]
You need CrossEntropy loss function instead of Logloss.

**Comment by halilbilgin** [comment]
Thanks.
